lib/utils/getSelectorMeaningfulSource.mjs

```diff
@@ -0,0 +1,45 @@
+import selectorParser from 'postcss-selector-parser';
+
+/**
+ * Get the source index of a selector, excluding leading whitespace.
+ *
+ * @param {import('postcss-selector-parser').Selector} node
+ *
+ * @returns {{selectorStr: string, index: number, endIndex: number}}
+ */
+export default function getSelectorMeaningfulSource(node) {
```

Comment:
- `getSelectorTrimmed`
- `getSelectorTrimmedSource`
- `getTrimmedSelectorSource`

?

Reply:
`getStrippedSelector`?

Reply:
`getStrippedSelectorSource` seems fine to me!
Thank you both for the suggestions 🙇